src/main/java/org/broadinstitute/hellbender/engine/filters/ReadGroupReadFilter.java

```diff
@@ -8,7 +8,7 @@
 import java.io.Serializable;
 
 /**
- * Only use reads from the specified read group.
+ * Keep only reads from the specified read group.
```

Comment:
Reads without RG does not have the specified string, so they are discarded. I think that it is clear enough with both sentences, but I will appreciate any suggestion to clarify that.